netlogo-gui/src/main/editor/Actions.scala

```diff
@@ -38,6 +38,8 @@ object Actions {
   def quickHelpAction(colorizer: Colorizer, i18n: String => String) =
     new MyTextAction(i18n("tabs.code.rightclick.quickhelp"),
       e => e.getHelpTarget(e.getSelectionStart).foreach(t => colorizer.doHelp(e, t)))
+  def codeCompletionAction(codeCompletionPopup: CodeCompletionPopup) = new MyTextAction("code-completion",
```

Comment:
Doesn't look like the parameter is actually used here, could this just be `def codeCompletionAction = ...`?